docs/configuration-options.md

```diff
@@ -0,0 +1,107 @@
+| `reportFormat`                      | Defines the benchmark report's format options.                                                                                                          | `"json"`(default), `"csv"`, `"scsv"`, `"text"`             | -                                                     |
+
+The following values can be used for specifying time unit:
+- "NANOSECONDS", "ns", "nanos"
+- "MICROSECONDS", "us", "micros"
+- "MILLISECONDS", "ms", "millis"
+- "SECONDS", "s", "sec"
+- "MINUTES", "m", "min"
+
+## Platform-Specific Configuration Options
+
+The options listed in the following sections allow you to tailor the benchmark execution behavior for specific platforms:
+
+### Kotlin/Native
+| Option                                        | Description                                                                                                            | Possible Values                    | Default Value    |
+|-----------------------------------------------|------------------------------------------------------------------------------------------------------------------------|------------------------------------|------------------|
+| `advanced("nativeFork", "value")`             | Executes iterations within the same process ("perBenchmark") or each iteration in a separate process ("perIteration"). | `"perBenchmark"`, `"perIteration"` | `"perBenchmark"` |
+| `advanced("nativeGCAfterIteration", value)`   | Whether to trigger garbage collection after each iteration.                                                            | `true`, `false`                    | `false`          |
+
+### Kotlin/JVM
+| Option                                      | Description                                                | Possible Values                        | Default Value  |
+|---------------------------------------------|------------------------------------------------------------|----------------------------------------|----------------|
+| `advanced("jvmForks", value)`               | Specifies the number of times the harness should fork.     | Non-negative Integer, `"definedByJmh"` | `1`            |
+
+**Notes on "jvmForks":**
+- **0** - "no fork", i.e., no subprocesses are forked to run benchmarks.
+- A positive integer value – the amount used for all benchmarks in this configuration.
+- **"definedByJmh"** – Let JMH determine the amount, using the value in the [`@Fork` annotation](https://javadoc.io/static/org.openjdk.jmh/jmh-core/1.21/org/openjdk/jmh/annotations/Fork.html) for the benchmark function or its enclosing class. If not specified by `@Fork`, it defaults to [Defaults.MEASUREMENT_FORKS (`5`)](https://javadoc.io/static/org.openjdk.jmh/jmh-core/1.21/org/openjdk/jmh/runner/Defaults.html#MEASUREMENT_FORKS).
```

Comment:
```suggestion
- **"definedByJmh"** – Let JMH determine the amount, using the value in the [`@Fork` annotation](https://javadoc.io/doc/org.openjdk.jmh/jmh-core/latest/org/openjdk/jmh/annotations/Fork.html) for the benchmark function or its enclosing class. If not specified by `@Fork`, it defaults to [Defaults.MEASUREMENT_FORKS (`5`)](https://javadoc.io/doc/org.openjdk.jmh/jmh-core/latest/org/openjdk/jmh/runner/Defaults.html#MEASUREMENT_FORKS).
```

Although, it might be worth using links to `1.21` as it's a version the plugin is compiled with.

Reply:
I think it's better to reference the latest version. It might contain improved documentation. I highly doubt the fundamental behavior of the annotation changes release to release